Library/Homebrew/cmd/search.rb

```diff
@@ -1,14 +1,17 @@
-#:  * `search`, `-S`:
+#:  * `search`:
 #:    Display all locally available formulae for brewing (including tapped ones).
 #:    No online search is performed if called without arguments.
 #:
-#:  * `search`, `-S` <text>|`/`<text>`/`:
+#:  * `search` (`--desc`) <text>|`/`<text>`/`:
```

Comment:
Parentheses (in contrast to square brackets) are used to denote mandatory arguments. Thus, they only make sense when there are multiple alternatives of which exactly one needs to be used. You probably want to use square brackets here and we'll need to document the effect of that switch. (Check out other commands in our man page for how that typically looks like.)